src/ppx_deriving_qcheck/args.ml

```diff
@@ -0,0 +1,24 @@
+open Ppxlib
+
+(** [curry_args args body] adds parameter to [body]
+
+    e.g.:
+    curry_args [gen_a; gen_b] () => fun gen_a -> fun gen_b -> ()
+*)
+let rec curry_args ~loc args body =
+  match args with
+  | [] -> body
+  | x :: xs -> [%expr fun [%p x] -> [%e curry_args ~loc xs body]]
+
+(** [apply_args args body] applies parameters to [body]
+
+    e.g.:
+    curry_args [gen_a; gen_b] f => f gen_a gen_b
```

Comment:
minor typo: `apply_args`

Reply:
Fixed in 4a25b84dd0811e5171b131482c1f60822527ecab.